charts/sftd/values.yaml

```diff
@@ -108,6 +108,10 @@ multiSFT:
   secret:
   turnServerURI:
 
+# This is the secret used for authentication when the multiSFT feature is disabled
+# Do not use this when multiSFT is enabled, and instead use the `multiSFT.secret` field
```

Comment:
How do `sftTokenSecret` and `multiSFT.secret` technically differ? If they are the same, it might be worth to consider having one field (instead of two) for the secret. This may come with some migration costs, though. :thinking:

Reply:
they are same as @z-dule mentioned to me. I found some env in Cailleach which are still using multiSFT.secret variable, removing it directly can be an issue.
We can add this new var and migrate all env to use this new var and later remove the multiSFT.secret variable.